config/confighttp/confighttp.go

```diff
@@ -216,6 +216,10 @@ type HTTPServerSettings struct {
 	// IncludeMetadata propagates the client metadata from the incoming requests to the downstream consumers
 	// Experimental: *NOTE* this option is subject to change or removal in the future.
 	IncludeMetadata bool `mapstructure:"include_metadata"`
+
+	// Additional headers attached to each HTTP response sent to the client.
+	// Header values are opaque since they may be sensitive.
+	Headers map[string]configopaque.String `mapstructure:"headers"`
```

Comment:
Should it be response_headers to clarify?

Reply:
I’m not opposed to it. Since HTTPClientSettings is using `headers` rather than `request _headers`, I went with that. It can be changed easily if that’s preferred.

Reply:
Indeed we don't use `request_headers`, but the request headers are the most common, that's kind of my thinking... On the server I can see a functionality to say route by this headers (accept requests only if these request headers are present) so I can see some usages for "headers". @open-telemetry/collector-approvers any idea?

Reply:
@bogdandrutu I'm not sure if the `@open-telemetry/collector-approvers` ping worked, but do let me know and I can revisit this

Reply:
I'd also lean towards `response_headers` even if it's inconsistent with the `HTTPClientSettings`'s `headers` (we can consider renaming them as well, but I'm not sure it's necessary). My reasoning is that request headers are more commonly discussed and come to mind first if referred by HTTP headers.

Reply:
Sounds good! I’ll make the change.